### PR TITLE
Merge amp handlers correctly

### PR DESCRIPTION
**Pull request by tochwill**
Resolves #NUMBER

**Overall change:** We were trying to compose two tap handlers together incorrectly, in the format `tap:action1;tap:action2` - it needs to be in the format `tap:action1,action2`. This also adds an id onto the AMP scrollable nav, so it can have its visibility toggled by an action.

**Code changes:**

- _A bullet point list of key code changes that have been made._
- _When describing code changes, try to communicate **how** and **why** you implemented something a specific way, not just **what** has changed._

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-navigation/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 6.0.0-alpha.9 | [PR#xxxx](https://github.com/bbc/psammead/pull/xxxx) Merge AMP tap handlers correctly |
```
Comment (tochwill) [inline packages/components/psammead-navigation/CHANGELOG.md]:
```suggestion
| 6.0.0-alpha.9 | [PR#2779](https://github.com/bbc/psammead/pull/2779) Merge AMP tap handlers correctly |
```

File: packages/components/psammead-navigation/README.md [hunk 2]
```diff
@@ -65,6 +65,7 @@ The `@bbc/psammead-navigation` package is a set of two components, `NavigationUl
 | -------- | ---- | -------- | ------- | ------- |
 | children | node | Yes      | N/A     | `<NavigationUl><NavigationLi url="/" script={latin} active="true">Home</NavigationLi><NavigationLi url="/sport" script={latin}>{Sport}</NavigationLi></NavigationUl>` |
 | dir      | string  | No       | `'ltr'`   | `'rtl'` |
+| id      | string  | No       | `''`   | `'scrollable-nav'` |
```
Comment (tochwill) [inline packages/components/psammead-navigation/README.md]:
```suggestion
| id      | string  | No       | `null`   | `'scrollable-nav'` |
```

File: packages/components/psammead-navigation/README.md [hunk 3]
```diff
@@ -195,7 +204,7 @@ import { latin } from '@bbc/gel-foundations/scripts';
   announcedText="Menu"
   onToggle="tap:menu.toggleVisibility"
```
Comment (tochwill) [inline packages/components/psammead-navigation/README.md]:
```suggestion
  onToggle="menu.toggleVisibility"
```

**Review by AlistairGempf — APPROVED** [review]
Looks good.

**Review by Bopchy — APPROVED** [review]

File: packages/components/psammead-navigation/README.md [hunk 4]
```diff
@@ -146,26 +147,30 @@ import { latin } from '@bbc/gel-foundations/scripts';
...
+      currentPageText="Current page"
     >
       Home
     </DropdownLi>
     <DropdownLi script={latin} service="news" key="Sport" url="/sport">
       Sport
     </DropdownLi>
   </DropdownUl>
-</Dropdown>
+</Dropdown>;
```
Comment (DenisHdz) [inline packages/components/psammead-navigation/README.md]:
```suggestion
</Dropdown>
```

File: packages/components/psammead-navigation/README.md [hunk 5]
```diff
@@ -175,13 +180,17 @@ import React from 'react';
...
+  announcedText="Menu"
+  isOpen={true}
+  onOpen={() => {
+    console.log('Handle open action');
+  }}
+  onClose={() => {
+    console.log('Handle close action');
+  }}
+  script={latin}
+/>;
```
Comment (DenisHdz) [inline packages/components/psammead-navigation/README.md]:
```suggestion
/>
```

File: packages/components/psammead-navigation/README.md [hunk 6]
```diff
@@ -193,9 +202,9 @@ import { latin } from '@bbc/gel-foundations/scripts';
 
 <AmpMenuButton
   announcedText="Menu"
-  onToggle="tap:menu.toggleVisibility"
+  onToggle="menu.toggleVisibility"
   script={latin}
-/>
+/>;
```
Comment (DenisHdz) [inline packages/components/psammead-navigation/README.md]:
```suggestion
/>
```

File: packages/components/psammead-navigation/README.md [hunk 7]
```diff
@@ -175,13 +180,17 @@ import React from 'react';
...
-    isOpen={true}
-    onOpen={() => { console.log("Handle open action"); }}
-    onClose={() => { console.log("Handle close action"); }} 
-    script={latin}
-/>
+<CanonicalMenuButton
+  announcedText="Menu"
+  isOpen={true}
+  onOpen={() => {
+    console.log('Handle open action');
```
Comment (DenisHdz) [inline packages/components/psammead-navigation/README.md:187]: Do you need this for testing atm?

File: packages/components/psammead-navigation/README.md (hunk 4 above)
Comment (tochwill) [reply packages/components/psammead-navigation/README.md]: Hmm... Prettier decided to add all of these in

File: packages/components/psammead-navigation/README.md (hunk 7 above)
Comment (tochwill) [reply packages/components/psammead-navigation/README.md:187]: I think this was just in the README to show that you need to pass a function in there, prettier just reformatted the file when I updated the README

**Review by DenisHdz — APPROVED** [review]

**Pull request merged**